index.html

```diff
@@ -107,52 +158,83 @@ <h2>About</h2>
+          <h3>Skills</h3>
+          <ul>
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>GitHub</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
+          <img src="images/icones/ic_skills.png" alt="htag">
+        </section>
+        <section class="technologies-section desk-deco">
+          <img class="shape" src="images/Shape.png" alt="shape">
+          <img class="imag-dot" src="images/image_dots.png" alt="htag">
+        </section>
       </section>
     </section>
     <section class="contact-form">
-      <img src="images/image_geometry_4.png" alt="">
+      <img class="contact-form-img-desk image-geometry-6" src="images/image-geometry_6.png" alt="">
+      <img class="contact-form-img-mobile" src="images/image_geometry_4.png" alt="">
       <header>
         <h2>Get started</h2>
         <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
           sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
       </header>
       <form action="https://formspree.io/f/xeqnzdea" method="POST">
-          <input type="text" name="name" placeholder="Your name"  maxlength="10" required>
-          <input type="email" name="email" placeholder="Your email" required>
-          <textarea name="message" placeholder="Enter your message here..." maxlength="500" required></textarea>
+        <div class="form-input">
+          <div class="field-name">
+            <label for="name">Your name</label>
+            <input type="text" name="name" id="name" placeholder="Your name"  maxlength="10" required>
+          </div>
+          <div class="field-email">
+            <label for="email"> Your email</label>
+            <input type="email" name="email" id="email" placeholder="Your email" required>
+          </div>
+        </div>
+        <div class="fiel-maessage">
+          <label for="message">Message</label>
+          <textarea name="message" id="message" placeholder="Enter your message here..." maxlength="500" required></textarea>
+        </div>
         <button type="submit">Start collaboration</button>
       </form>
+      <img class="contact-form-img-desk image-geometry_7" src="images/image-geometry_7.png" alt="">
     </section>
+    <footer>
+      <p class="mobile-resume-button"> Get my resume    <i class="fa-solid fa-up-right-from-square"></i></p>
+      <p class="desk-resume-button"> Get my CV    <i class="fa-solid fa-download"></i></p>
+      <div class="social-icones">
+        <a href="https://stackoverflow.com/"><i class="fa-brands fa-stack-overflow"></i></a>
+        <a href="https://github.com/Baroka-wp/"><i class="fa-brands fa-github"></i></a>
+        <a href="https://twitter.com/IrotoriB"><i class="fa-brands fa-twitter"></i></a>
+        <a href="https://www.linkedin.com/in/baroka/"><i class="fa-brands fa-linkedin"></i></a>
+        <a href="https://medium.com/"><i class="fa-brands fa-medium"></i></a>
```

Comment:
### OPTIONAL 

- [ ] Kindly add the appropriate icon, so your design looks exactly like the Figma design 🤝 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![vvv](https://user-images.githubusercontent.com/69638013/159924182-199b2ed8-055b-48c2-b7b9-dce1c6e23071.png) | ![Screenshot_29](https://user-images.githubusercontent.com/69638013/159923786-f80625b9-5e6d-4973-8114-a658c1e716df.png)